Start.tsx

```diff
@@ -162,57 +145,32 @@ function Start({ onReady }: StartProps) {
       dispatch(setSetting({ theme: usedTheme }));
     }
 
-    const newTheme = extendTheme(
-      merge.all([
-        ThemeOptionsMap[usedTheme],
-        accentColor
-          ? {
-              colors: {
-                app: {
-                  accent: accentColor,
-                },
-              },
-            }
-          : {},
-        {
-          components: {
-            Text: {
-              defaultProps: {
-                color: ThemeOptionsMap[usedTheme].colors.app.textPrimary,
-              },
-            },
-          },
-        },
-        isSystemTextSize
-          ? {
-              components: {
-                Text: {
-                  defaultProps: {
-                    allowFontScaling: false,
-                  },
-                },
-              },
-            }
-          : { fontSizes: getFontScale() },
-        isSystemFont ? systemFontSettings : {},
-      ])
-    );
-    // TODO add fallback
-    setSelectedTheme(newTheme);
+    // Disabling Font Scaling for now
+    // const newTheme = merge.all([
+    //   isSystemTextSize
+    //     ? {
+    //         components: {
+    //           Text: {
+    //             defaultProps: {
+    //               allowFontScaling: false,
+    //             },
+    //           },
+    //         },
+    //       }
+    //     : { fontSizes: getFontScale() },
+    //   isSystemFont ? systemFontSettings : {},
+    // ]);
+
+    // // TODO add fallback
     setStatusBarColor(
-      newTheme.config.initialColorMode === "dark" ? "light" : "dark"
+      ThemeOptionsMap[usedTheme].config.initialColorMode === "dark"
+        ? "light"
+        : "dark"
     );
 
-    setRootViewBackgroundColor(ThemeOptionsMap[usedTheme].colors.app.bg);
+    setRootViewBackgroundColor(ThemeOptionsMap[usedTheme].colors.bg);
     // ! fontSize has to be here
-  }, [
-    currentTheme,
-    fontSize,
-    getFontScale,
-    isSystemTextSize,
-    isSystemFont,
-    accentColor,
-  ]);
+  }, [currentTheme, fontSize, getFontScale, isSystemFont, accentColor]);
```

Comment:
Probably not. With font scaling commented out, it won't make a difference since the value shouldn't change, so it's just a matter of cleanup.  It sounded like @gkasdorf was going to try uncomment the font scaling though.

Reply:
I'm gonna try to work on font scaling at some point soon, can we at least leave a todo there so i remember to re-add it?